test-display-runner-using-downloaded-installer.js

```
@@ -2,16 +2,28 @@ global.log = console;
 const displayId = process.argv[2];
 const numberOfPrints = process.argv[3];
 
+const platform = require("rise-common-electron").platform;
+const path = require("path");
+const fs = require("fs");
+
+const launcherUtils = require("./utils/launcher-utils.js");
 const presentation = require("./presentation");
 const installerStarter = require("./installer-starter");
 
+const preparePlayerModule = function () {
+  try {fs.mkdirSync(path.join(launcherUtils.getInstallDir(), "modules", "player-electron"));} catch(err) {};
```

@andrecardoso is it safe to ignore errors thrown on this code ?

---

Good question. I copied this line from https://github.com/Rise-Vision/rise-launcher-electron-e2e/blob/master/clean-previous.js#L19 I think we can just let the script fail here

---

Actually.. I think the try/catch is there to not fail when the directory already exists. I'll add some code to check if and avoid failing.